source/IOSResolver/src/IOSResolver.cs

```diff
@@ -125,11 +125,15 @@ private class Pod {
         /// <summary>
         /// Get the path of a pod without quotes.  If the path isn't present, returns an empty
         /// string.
+        /// This also support podspec parameters. It will prefer podspec parameter over path 
+        /// if present.
         /// </summary>
         public string LocalPath {
             get {
                 string path;
-                if (!propertiesByName.TryGetValue("path", out path)) return "";
+                if (!propertiesByName.TryGetValue("podspec", out path)){
```

Comment:
It is better to use a different property for "podspec".

Also, do similar override in `PodFilePodLine`
https://github.com/camphongdinh/unity-jar-resolver/commit/a97b71d71ef373adb64c5688fa4f00a620189545#diff-4e150db5d98cbbe99ab461893cc478b33377658b810b34ebc72f7549670ef482R154-R157

Note that from the Cocoapod doc, ":podspec" can point to an "http" url.  So perhaps only expand the path only if it does not start with "http".